politeiawww/backend.go

```diff
@@ -737,8 +766,13 @@ func (b *backend) ProcessNewProposal(np www.NewProposal) (*www.NewProposalReply,
 		return nil, err
 	}
 
+	name, err := getProposalName(np.Files)
+	if err != nil {
+		return nil, err
+	}
+
 	n := pd.New{
-		Name:      sanitize.Name(np.Name),
+		Name:      name,
```

Comment:
We probably should kill this in D as well.

Reply:
killed + dep ensure + post action added to description